docs/react-native/quick-setup.md

```diff
@@ -242,7 +242,3 @@ const styles = StyleSheet.create({
   },
 });
 ```
-
-### More
-
```

Comment:
**note:** I deleted it because it was linking to the same category menu. Users can also navigate to the next page.